doc/directives.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018-2019 Xanadu Quantum Technologies Inc.
```

Comment:
```suggestion
# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
```

Reply:
This seems to be the case for many sources and test files as well, a separate PR could change this systematically (e.g. by using `sed` from the command line).